### PR TITLE
WIP: Add table.c

### PR DESCRIPTION
**Pull request by radare**

### CODE REVIEW

**Comment by XVilka** [comment]
@deepakchethan please take a look, if you have any questions or a suggestions here.

**Comment by radare** [comment]
its still unfinished, but i expect to finish this before he is done with the issues he is working on now

File: libr/util/t/main.c [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#include "../table.c"
```
Comment (Maijin) [inline libr/util/t/main.c]: main repository is not the place for unit testing

Comment (Maijin) [reply libr/util/t/main.c]: https://github.com/radare/radare2-regressions/tree/master/unit

**Comment by radare** [comment]
Obv this wont be merged. Its just for testing purposes until the thing gets ready to be merged. Hopefully tonight

**Pull request merged**